gvm/connections.py

```diff
@@ -142,6 +142,11 @@ def disconnect(self):
         except OSError as e:
             logger.debug('Connection closing error: %s', e)
 
+    def finish_send(self):
+        """Indicate to the remote server you are done with sending data
+        """
+        self._socket.shutdown(1)
```

Comment:
I think using `socketlib.SHUT_WR` or another explanation of the parameter value would make the code even more readable.